content/chainguard/chainguard-images/how-to-use/verifying-chainguard-images-and-metadata-signatures-with-cosign/index.md

```diff
@@ -145,6 +175,20 @@ GitHub Workflow Ref: refs/heads/main
 ...
 ```
 
+
+### Private/Dedicated Registry
+
+```shell
+PARENT=your-registry-name
```

Comment:
You don't have this in the above and it will overwrite the defined variable, so I would remove.